src/Datasource/EntityTrait.php

```diff
@@ -894,21 +894,21 @@ public function isNew($new = null)
     /**
      * Returns whether this entity has errors.
      *
-     * @param bool $includeHasErrorsFromNestedEntities
+     * @param bool $includeNested true will check nested entities for hasErrors()
      * @return bool
      */
-    public function hasErrors($includeHasErrorsFromNestedEntities = true)
+    public function hasErrors($includeNested = true)
     {
-        if (empty($this->_errors) === false) {
+        if (!empty($this->_errors)) {
             return true;
         }
 
-        if ($includeHasErrorsFromNestedEntities === false) {
+        if ($includeNested === false) {
```

Comment:
`!$includeNested`

Reply:
Isn't it better to use `=== false` in this case to be more strict about the type of the value?
Various other if statements in the Entity class also use `=== false`.